macros/src/main/scala/com/gu/contentapi/circe/CirceScroogeMacros.scala

```diff
@@ -109,7 +109,7 @@ object CirceScroogeMacros {
     _root_.io.circe.Decoder.instance((cursor: _root_.io.circe.HCursor) => {
       cursor.focus.asString match {
         case _root_.scala.Some(value) =>
-          val withoutHyphens = value.replaceAllLiterally("-", "")
+          val withoutHyphens = org.apache.commons.lang3.StringUtils.remove(value, '-')
```

Comment:
@mchv Could you add `commons-lang3` as a dependency? At the moment it's only available by chance as a transitivite dep of `libthrift`.